### DISCUSSION TITLE
Troubleshooting using an existing SMTP email provider

### DISCUSSION THREAD

**Discussion by RustyXP** [mozilla/hubs#2796] [General Questions & Discussion] [answered]
Hello Hubs Team!

I have Hubs Cloud up and running on AWS, however when I add my SMTP email details to the Email tab under Server Settings, it's not sending the magic link emails via my existing email provider.

I can't see any record of Hubs trying to access my SMTP server, so I suspect the issue is on the Hubs side. Is there a way to test or monitor email activities on the Hubs server? Any recommendations on other things to check or test? In case it helps to know, I'm using Dreamhost for email. Dreamhost uses port 465 for SMTP.

Thanks for your guidance.

**Comment by robinkwilson** [comment, marked as answer]
Good question, we're seeing more of the troubleshooting an existing SMTP email provider logged and we need to look into them. 

Here's the [hubs-cloud issue](https://github.com/mozilla/hubs-cloud/issues/95) to track progress.
